cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CouponDTO.java

```diff
@@ -250,6 +250,10 @@ public boolean isLicenseOnly()
         return licenseOnly;
     }
 
+    /**
+     * @deprecated use {@link #setDestination(CouponDestination)}
+     */
+    @Deprecated
     public void setLicenseOnly( boolean licenseOnly )
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)